tests/export.bats

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "extract" {
+  touch ${TESTDIR}/reference-time-file
+  for source in scratch alpine; do
+    cid=$(buildah from --pull=true --signature-policy ${TESTSDIR}/policy.json ${source})
+    mnt=$(buildah mount $cid)
+    touch ${mnt}/export.file
+    buildah umount $cid
+    buildah export "$cid" > ${TESTDIR}/${source}.tar
+    buildah export -o ${TESTDIR}/${source}1.tar "$cid"
```

Comment:
Can we add a `run tar tf` on one of these and check that the output matches the expected contents of the tarball?